### ISSUE TITLE
Using override_weapons.xml file removes Telescoping Staff

### ISSUE THREAD

**Issue by Thelinil** [chummer5a/chummer5a#1539]
### Environment
Chummer Version:  Nightly-v5.191.70
Environment: Windows 8.1
Runtime: .NET 4.5.1

### Expected behaviour
First placing an override_weapons.xml file into the 'Data' folder. Once a character is loaded, the Telescoping Staff should appear under the 'Clubs' drop down in the weapon select menu.

### Actual behaviour
It does not appear.

### Steps to reproduce the behaviour
Create an override_weapons.xml file and place it into the 'Data' folder.

### Link to affected character file
This is persistent across character files.

I checked first to see if it was an issue with the edits I made in my override_weapons.xml file. I created a new override_weapons.xml file which was identical copy of the weapons.xml file (except for the name), which still caused the Telescoping Staff to not appear. I was thinking this might have something to do with the Telescoping Staff and the Stun Dongle sharing a GUID, b4b3a872-1532-4477-9bf3-76db34dc7bbf. Yet this issue doesn't appear until the override_weapons.xml file is introduced, and adding custom_weapons.xml files don't cause this behavior either.

**Comment by chummer5a** [comment]
Hm, I can't replicate this. Can you show me the override file you're using?

**Comment by Thelinil** [comment]
I was able to replicate this on the same computer using a fresh copy of the
latest nightly, 5.191.75, using an override_weapons file. I'll try on a
different computer.

**Comment by Thelinil** [comment]
I was able to replicate this on my buddy's Windows 10 laptop, using the
latest nightly as well.

**Comment by chummer5a** [comment]
What are you overriding though? I've tested it with multiple variations of a weapon and can't get it to misbehave.

**Comment by Thelinil** [comment]
I don't think the particular changes I've made in my override file matters.
I've been able to cause this behavior with both my override_weapons.xml
file (with a bunch of small changes) and a new override_weapons.xml file
(which is identical to the 'stock' weapons.xml file except for the name).
Both will cause the Telescoping Staff to disappear from the drop-down menu.
I've tested this on the new Nightly and the same issue occurs.
I've also been testing this a fresh version of chummer, so there aren't any
houserule options or anything like that could be affecting behavior outside
of the override file.
